vm-image-spec.yaml

```diff
@@ -377,13 +392,19 @@ files:
         query: |
           select pg_size_bytes(current_setting('neon.file_cache_size_limit')) as lfc_cache_size_limit;
 
-      - metric_name: lfc_approximate_working_set_size
+      - metric_name: lfc_approximate_working_set_size_seconds
         type: gauge
         help: 'Approximate working set size in pages of 8192 bytes'
-        key_labels:
-        values: [approximate_working_set_size]
+        key_labels: [duration_minutes]
+        values: [size]
+        # NOTE: This is the "internal" / "machine-readable" version. This outputs the working set
+        # size looking back 1..60 minutes, labeled with the number of minutes.
         query: |
-          select neon.approximate_working_set_size(false) as approximate_working_set_size;
+          select
+            x::text as duration_minutes,
+            neon.approximate_working_set_size_seconds(60 * x) as size
```

Comment:
So, this `neon.approximate_working_set_size_seconds` is able to take any `duration` in range `[0, neon.wss_max_duration]`, and will return a WSS estimation on a window `[now - duration, now]`, right?

Reply:
Correct, yeah